source/playgrounds.txt

```diff
@@ -177,6 +181,26 @@ of your playground.
 immediately above your selected section. Click this link to test and
 troubleshoot specific lines or sections of your playground.
 
+Code Formatting and Linting Tools
+---------------------------------
+
+MongoDB Playgrounds are proper JavaScript files and can work with popular
+|vscode-short| formatting and linting extensions like
+`Prettier <https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode>`_ 
+and `ESLint <https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint>`_.
+
+If you are using a code formatting extension, MongoDB Playgrounds 
+suggests autocompletion and liniting hints for:
+
+- System variables such as ``$$ROOT`` and ``$$NOW``.
+- ``use`` and ``db`` commands.
+- Collection names, in bracketed notation such as ``db["collection"].find({})``.
+- Alternative commands for shell commands not supported in MongoDB Playgrounds 
```

Comment:
Is the "use database" example in the last bullet repeating the same idea in the second bullet, where we say "use commands"?

Maybe in the last bullet we remove `use database` and only mention `show users` as an example, to avoid repetition.

Reply:
Yes, it is, removed the last bullet example for use database.